src/main/java/com/minecolonies/rankup/modules/core/config/GroupsConfig.java

```diff
@@ -47,7 +48,7 @@
         @Setting(value = "timing-rankdown", comment = "If this value is true, if a player is within this group and they DO NOT have the require play time, they will be ranked down")
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)